near-sdk-sim/Cargo.toml

```diff
@@ -21,6 +21,8 @@ near-vm-logic = "=4.0.0-pre.1"
 near-pool = "=0.1.0-pre.1"
 near-store = "=0.1.0-pre.1"
 near-runtime = "=4.0.0-pre.1"
+serde = {version = "1", features=["derive"]}
```

Comment:
Hm, wait, are we actually using serde here?

Reply:
Hmm i thought i need this to use serde_json, not?

Reply:
Not really, no. You only need serde if you want to `derive(Deserialize)`. For `serde_json::from_slice`, you only need `serde_json`.

Reply:
:+! ah that's precise!